core/src/components/cat-tabs/cat-tabs.scss

```diff
@@ -0,0 +1,32 @@
+@use 'variables' as *;
+@use 'mixins' as *;
+
+:host {
+  display: flex;
+  flex-direction: row;
+}
+
+:host([hidden]) {
+  display: none;
+}
+
+.tab {
+  box-shadow: inset 0 -1px 0 0 cat-token('color.ui.border.tab');
+  padding: 0.5rem;
+}
+
+.tab-active {
+  box-shadow: inset 0 -3px 0 0 cat-token('color.ui.border.activeTab');
+}
+
+.tab-align-left::part(button) {
+  justify-content: start;
```

Comment:
there is a misunderstanding what the tab alignment should look like. Here you can see an example of left and center aligned tabs: https://stackblitz.com/run?file=src%2Fapp%2Ftab-group-align-example.ts
Here is an example of justified tabs: https://stackblitz.com/run?file=src%2Fapp%2Ftab-group-stretched-example.ts

so the alignment is applied to the tabs and not to the content of the tabs. makes sense? Please get in touch if this is still unclear. If you want, you can also add a right alignment option (as seen in the first example). But this would be more for the sake of completeness